src/snovault/elasticsearch/esstorage.py

```diff
@@ -80,11 +80,17 @@ def storage(self):
             return self.read
         return self.write
 
+    def force_database(self):
```

Comment:
We're not using the self reference.   @staticmethod should be applied and the self removed.

Reply:
Fair point, but I just turned it into a function at module scope since there's no particular reason it belongs in the PickStorage class.